SportsHub/SportsHub.DAL/Repository/ArticleRepository.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.EntityFrameworkCore;
+using SportsHub.DAL.Data;
+using SportsHub.Domain.Models;
+using SportsHub.Domain.Repository;
+
+namespace SportsHub.DAL.Repository
+{
+    public class ArticleRepository : GenericRepository<Article>, IArticleRepository
+    {
+        public ArticleRepository(ApplicationDbContext context) : base(context)
+        {
+        }
+
+        public async Task<Article?> GetByTitleAsync(string title)
+        {
+            return await DbSet
+                .Where(x => x.Title == title)
+                .FirstOrDefaultAsync();
+        }
+
+        public async Task<IEnumerable<Article>> GetByAuthorAsync(string author)
+        {
+            return await DbSet
+                .Where(x => x.Authors.Any(a => a.DisplayName == author))
```

Comment:
Let's discuss this feature later. Is it requirement for getting an article by DisplayName?